qiskit/transpiler/passes/routing/stochastic_swap.py

```diff
@@ -43,7 +49,9 @@ class StochasticSwap(TransformationPass):
            the circuit.
     """
 
-    def __init__(self, coupling_map, trials=20, seed=None, fake_run=False):
+    _instance_num = 0  # track number of instances of this class
```

Comment:
Using the "+1" is actually a little worse because it will use that same number every time a controlflow instruction is encountered as dag at that instance depth.

Reply:
Done in b34a101637dc19dd112ebe0cbc45d232c37cdbb2.